melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/mottak/SedMottakService.java

```diff
@@ -42,6 +42,8 @@ public void behandleSed(SedMottattHendelse sedMottattHendelse) {
             return;
         }
 
+        final var sed = euxService.hentSedMedRetry(sedMottattHendelse.getSedHendelse().getRinaSakId(),
+            sedMottattHendelse.getSedHendelse().getRinaDokumentId());
         sedMottattHendelseRepository.save(lagretHendelse);
```

Comment:
Dette er vel bare for å oppdatere journalpostID så det kunne flyttes i `try catch`?